### ISSUE TITLE
Hub/Calendar/Contacts links not keyboard-accessible

### ISSUE THREAD

**Issue by ndarilek** [soapdog/patchfox#76]
Hello, blind screen reader user here. Thanks for Patchfox. Tried it out a while back, but having to reload the source folder as an extension was a blocker.

I notice that the links to the hub, composition screen, etc. can't be triggered by Enter. I can tab between them just fine, but nothing I do seems to make them appear.

If I tab to Compose, there does appear to be a Post item that I can activate via Enter, but that's about it.

Just skimmed your code. Maybe the global menu just needs a keydown handler for Enter on the links?

Thanks again.

**Comment by soapdog** [comment]
@ndarilek thanks a lot for raising this issue. I'll try to improve the accessibility of patchfox, in the meanwhile, I suspect you might be better served by using [Oasis](https://github.com/fraction/oasis/) which has many advantages over Patchfox for your use case. First, it doesn't use any client-side Javascript, so from the screen reader point of view it should be just simple, well structured HTML. Because of that, all the keyboard triggers should work as well.

**Comment by soapdog** [comment]
I'm trying to fix that but apparently macOS makes it's own accessibility decision on what should be focusable. At the moment I'm a bit out of my depth. I'm trying to improve accessibility on Patchfox but I can't commit for a date to have this working correctly, mostly because it involves me learning what to do. 

The recommendation above, Oasis, should work much better and I think it is a better option than Patchfox for users browsing through screen readers.

I just pushed a commit with some changes to the `globalMenu` package that should improve accessibility. I follow the instructions about it from MDN. I don't think it is working since I can't focus on them using the keyboard. I've learned that macOS has some accessibility quirks and will not allow all controls to be focused with a keyboard by default.

If you have the repository cloned over there, a pull and rebuild should get you working with the improved menu. I'd be keen to learn if it is working better.

My recommendation to use Oasis stands though, I rather have screen reader users switch to a product better suited to their uses than have them have a poor experience with Patchfox while I sort this out.

**Comment by soapdog** [comment]
I received no feedback on this after the comments, so I'm closing it. 

I'm taking care to improve accessibility in Patchfox, but I still believe that screen-reader users would be better used by Oasis.
